docs/configuration.md

```diff
@@ -740,6 +740,20 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.eventLog.overwrite</code></td>
+  <td>false</td>
+  <td>
+    Whether to overwrite any existing files.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.eventLog.buffer.kb</code></td>
+  <td>100</td>
+  <td>
+    Buffer size to use when writing to output streams.Buffer size in KB.
```

Comment:
Might want to proofread these before pinging for review. You're missing a space before the second sentence, but, why not just: "Buffer size in KB to use when writing to output streams"

Reply:
I have fixed the description and correction unit.
Please check org.apache.spark.internal.config#EVENT_LOG_OUTPUT_BUFFER_SIZE

  private[spark] val EVENT_LOG_OUTPUT_BUFFER_SIZE = ConfigBuilder("spark.eventLog.buffer.kb")
    .bytesConf(ByteUnit.KiB)
    .createWithDefaultString("100k")